src/main/java/gov/nih/nci/evs/api/controller/SubsetController.java

```diff
@@ -107,14 +106,12 @@ public class SubsetController extends BaseController {
       method = RequestMethod.GET,
       value = "/subset/{terminology}",
       produces = "application/json")
```

Comment:
This was being cached to address a performance problem we don't have anymore.  And it was breaking tests the way it was configured.  Keeping it commented out just in case we need it.